aggregator/src/chunk.rs

```diff
@@ -0,0 +1,60 @@
+//! This module implements `Chunk` related data types.
+//! A chunk is a list of blocks.
+use eth_types::H256;
+use ethers_core::utils::keccak256;
+
+#[derive(Default, Debug, Clone)]
+/// A chunk is a set of continuous blocks.
+/// A ChunkHash consists of 4 hashes, representing the changes incurred by this chunk of blocks:
+/// - state root before this chunk
+/// - state root after this chunk
+/// - the withdraw root of this chunk
+/// - the data hash of this chunk
+pub struct ChunkHash {
+    /// Chain identifier
+    pub(crate) chain_id: u8,
```

Comment:
sorry, made a mistake before. `chain_id` should be `u32` type instead of `u8`.

Reply:
fixed with a281f267fd00930e7322bfe47efec2e3a91a21bf